junit-platform-commons/src/module/org.junit.platform.commons/module-info.java

```diff
@@ -9,7 +9,8 @@
  */
 
 module org.junit.platform.commons {
-	requires java.logging; // TODO Is "requires transitive java.logging" needed here?
+	requires java.logging;
```

Comment:
I left the `static` modifier out on purpose -- because we need the module when running on the module path.

Otherwise, users have to add it via `--add-module java.management` or via their module descriptor(s).